lib/bap_types/bap_bitvector.ml

```diff
@@ -178,15 +178,15 @@ let pp_generic
     | `this x -> fprintf ppf "%s" x
     | `base -> pp_prefix ppf
     | `auto ->
-      if Bitvec_order.(x >= (min (word 10) base))
+      if Z.Compare.(x >= (Z.min (word 10) base))
```

Comment:
wow, it was a bug before... not that it was breaking before, but at least the code as written was not the code as meant.